src/common/utility.cpp

```diff
@@ -410,19 +388,7 @@ QString Utility::platformName()
 {
     QString re("Windows");
 
-#if QT_VERSION < QT_VERSION_CHECK(5, 4, 0)
-    if (isMac()) {
-        re = QLatin1String("MacOSX");
-    } else if (isLinux()) {
-        re = QLatin1String("Linux");
-    } else if (isBSD()) {
-        re = QLatin1String("BSD");
-    } else if (isUnix()) {
-        re = QLatin1String("Unix");
-    }
-#else
     re = QSysInfo::prettyProductName();
```

Comment:
could just be 
´´´
return QSysInfo::prettyProductName();
´´´

No need to initialize re to "Windows"